source/decisions/2023-09-25-support-djangocon-africa.rst

```diff
@@ -0,0 +1,18 @@
+2023-03-04 Support of DjangoCon Africa 2023
+===============================================
+
+Background
+----------
+
+DjangoCon Africa takes place on 6-11 of November 2023. On 7 September 2023 the DjangoCon
+Africa Chair contacted the trustees by email to request monetary support for the event.
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to 
+Silver level sponsorship.
+
+The exact mechanism of doing so (donation, grant, sponsorship) is considered an 
```

Comment:
We implicitly consider anything outside of the scope of the decision itself an implementation detail - I'm not sure this aspect needs explicitly calling out any more than e.g. whether we're paying by cheque or bank transfer - what do you think about dropping this?